src/client/store/ui/userManagement/actions/getUsersCount.ts

```diff
@@ -15,6 +15,7 @@ type Params = {
   cycleName: string
   countries?: Array<CountryIso>
   fullName?: string
+  includeRoleTotals?: boolean
```

Comment:
Yes, added it:

![image](https://github.com/user-attachments/assets/c67b771c-a3d9-4b9f-987b-f0abd43249bd)

Reply:
Yes, I was trying to avoid having the extra user counts in the table paginated store, but I guess it's no problem to have them there too as long as the property `total` is defined (otherwise the pagination wouldn't work).